### PR TITLE
feat: Add runner logfiles to output

### PR DESCRIPTION
**Pull request by sanketrathi**
This PR adds logfile information for runners to the module output

With this PR, we add propagate `runners.logfiles` out through the module
output. This would potentially enable module users to create CloudWatch
queries / metric filters (eg: runner startup time)

### CODE REVIEW

File: modules/multi-runner/outputs.tf [hunk 1]
```diff
@@ -10,6 +10,7 @@ output "runners" {
     role_scale_up           = runner.role_scale_up
     role_scale_down         = runner.role_scale_down
     role_pool               = runner.role_pool
+    logfiles                = module.runners.logfiles
```
Comment (npalm) [inline modules/multi-runner/outputs.tf:13]: This won't work, you need to refer to the runner object.
```suggestion
    logfiles                = runner.logfiles
```

**Review by npalm — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please can yo shortly describe the need for the output, would like to understand the problem you solving. In general ok, one small remark.

File: modules/multi-runner/outputs.tf (hunk 1 above)
Comment (sanketrathi) [reply modules/multi-runner/outputs.tf]: Thanks for the review @npalm . The PR got closed with an accidental force push, sincere apologies

Comment (sanketrathi) [reply modules/multi-runner/outputs.tf]: I've also updated the PR description with the motivating use case(s) I'm trying to solve with this change

Comment (sanketrathi) [reply modules/multi-runner/outputs.tf]: Another quick question: I was also planning on modifying the `multi-runner` module to allow `ami_id_ssm_parameter_name` as a parameter with `runner_config`. Should I raise a separate PR for it?

Comment (npalm) [reply modules/multi-runner/outputs.tf]: Yes please create a separate PR

File: modules/multi-runner/outputs.tf [hunk 2]
```diff
@@ -10,6 +10,7 @@ output "runners" {
     role_scale_up           = runner.role_scale_up
     role_scale_down         = runner.role_scale_down
     role_pool               = runner.role_pool
+    logfiles                = runners.logfiles
```
Comment (npalm) [inline modules/multi-runner/outputs.tf:13]:
```suggestion
    logfiles                = runner.logfiles
```

**Review by npalm — CHANGES_REQUESTED** [review]

**Review by npalm — APPROVED** [review]

**Review by npalm — APPROVED** [review]

**Pull request merged**